src/vcbuild/dmd.vcxproj

```
@@ -122,7 +122,7 @@
     </Link>
     <DCompile>
       <VersionIdentifiers>MARS</VersionIdentifiers>
-      <StringImportPaths>..\..\res;$(OutDir)</StringImportPaths>
+      <StringImportPaths>../dmd/res;$(OutDir)</StringImportPaths>

```

```suggestion
      <StringImportPaths>..\dmd\res;$(OutDir)</StringImportPaths>

```

(similar for the changes below)

This is the Visual Studio project file, so I am pretty sure the Windows slashes should be kept.